README.md

```diff
@@ -56,15 +58,15 @@ Learn how to automate your workflow using Twilio's REST API and Twilio SMS. This
    pip install -r requirements.txt
    ```
 
-1. Edit the following keys/values for the `config.py` file inside the  `airtng_flask/` directory. Be sure to replace the place holders and connection string with real information. Replace the connection string preferably under development config.
+1. Copy the sample configuration `.env.example` to `.env`, and then edit `.env` to match your configuration.
```

Comment:
I notice every step is numbered `1`. These should be fixed to go in ascending order